agent/command/git.go

```diff
@@ -254,7 +254,9 @@ func (c *gitFetchProject) buildCloneCommand(ctx context.Context, conf *internal.
 			// https://docs.github.com/en/rest/guides/getting-started-with-the-git-database-api#checking-mergeability-of-pull-requests
 			commitToTest, err = c.waitForMergeableCheck(ctx, opts.owner, opts.repo, conf.GithubPatchData.PRNumber)
 			if err != nil {
-				return nil, err
+				logger.Task().Error(errors.Wrap(err, "error checking if pull request is mergeable"))
```

Comment:
I don't think the agent should wrap.

Reply:
this one is to give more context on the error for the user. Currently it's just `error getting pull request data from Github: GET https://api.github.com/repos/10gen/dsi/pulls/875: 404 Not Found []`. I'd prefer to keep it

Reply:
SGTM